homeassistant/components/airzone/entity.py

```diff
@@ -67,6 +70,42 @@ def get_airzone_value(self, key: str) -> Any:
         return value
 
 
+class AirzoneWebServerEntity(AirzoneEntity):
+    """Define an Airzone WebServer entity."""
+
+    def __init__(
+        self,
+        coordinator: AirzoneUpdateCoordinator,
+        entry: ConfigEntry,
+    ) -> None:
+        """Initialize."""
+        super().__init__(coordinator)
+
+        self._attr_device_info: DeviceInfo = {
+            "connections": {
+                (
+                    dr.CONNECTION_NETWORK_MAC,
+                    self.get_airzone_value(AZD_MAC),
+                )
+            },
+            "identifiers": {(DOMAIN, f"{entry.entry_id}_ws")},
+            "manufacturer": MANUFACTURER,
+            "model": self.get_airzone_value(AZD_MODEL),
+            "name": self.get_airzone_value(AZD_FULL_NAME),
+            "sw_version": self.get_airzone_value(AZD_FIRMWARE),
+        }
+        self._attr_unique_id = (
+            entry.entry_id if entry.unique_id is None else entry.unique_id
+        )
```

Comment:
No, it shouldn't appear later since Local API features are checked on the first aioairzone .validate() or .update() and if webserver (which reports mac for unique id) is not supported, then consecutive calls won't try to fetch webserver data.

Reply:
> Will it ever appear after restart ? I'm think of the case where it isn't supported and than becomes supported where we need to migrate the unique ids since it could change out from under the entity

Yes, if the Airzone WebServer firmware is upgraded and restarted, and the integration is reloaded or Home Assistant is restarted, then it will appear and HA entities should be migrated from entry_id to unique_id.

Reply:
This is an existing problem.  It can be addressed in another PR.

`flux_led` has the exact same issue so you can likely copy most of the migration code from there.

Reply:
I'm a little bit lost here @bdraco...
Isn't that what you already requested in add7103d554a4f5564b73fc342238465ff86e689 https://github.com/home-assistant/core/pull/70287?
The unique ID migration was implemented in _async_migrate_unique_ids.
If it's something different I'm not getting what you mean.

Reply:
It is all good. I've seen too many PRs in between that one and this one and missed the fact you already did it.